packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721BurnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "./AttestationStation.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+/**
+ * @title  Optimist
```

Comment:
Why is this contract ownable? There is no usage of `onlyOwner`

Reply:
Ah i see the call to `owner` now

Reply:
Yea it's a little awkard actually.  Ideally we would do a check to make sure any owner being transferred to has the correct attestations but I held off on implementing that.